astropy/units/quantity_helper/function_helpers.py

```diff
@@ -702,8 +722,74 @@ def _check_range(range, unit):
     return range
 
 
-@function_helper
-def histogram(a, bins=10, range=None, weights=None, density=None):
+@function_helper(helps=np.histogram_bin_edges)
```

Comment:
Nitpick and not new, but no need to add `(helps=...)` if it is the same function as you're helping.